### PR TITLE
Update decorators.md

### PR DESCRIPTION
**Pull request by glrodasz**
Issue:

## What I did
I update the docs, to make clear that multiple decorators can be defined.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: docs/writing-stories/decorators.md [hunk 1]
```diff
@@ -155,3 +155,7 @@ All decorators relevant to a story will run in the following order once the stor
 - Global decorators, in the order they are defined
 - Component decorators, in the order they are defined
 - Story decorators, in the order they are defined
+
+## Multiple decorators
+
+You can define multiple decorators, but keep in mind that the last one in the list will wrap the story first, then the earlier ones will wrap those.
```
Comment (jonniebigodes) [inline docs/writing-stories/decorators.md:161]: @glrodasz If you're ok with it, I'd suggest a bit more explaining and if possible an example to demonstrate this behavior, to allow our community to have a bit more context on it What do you think?

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@glrodasz thanks for taking the time and effort into this pull request, appreciate it. I left a small item that needs your attention and I would love to hear your thoughts about it.

Let me know and we'll go from there.

File: docs/writing-stories/decorators.md (hunk 1 above)
Comment (glrodasz) [reply docs/writing-stories/decorators.md:161]: Sure! give me this week and I will improve it.

**Comment by jonniebigodes** [comment]
@glrodasz, just a friendly reminder to see if you're willing to continue to work on this so that we can merge it. Let me know, and we'll go from there.

Hope you have a great day.

Stay safe

**Comment by glrodasz** [comment]
Hey @jonniebigodes yes! I got a lot of work, and I also notice that I need to clone the repo and include the CodeSnippets, I will give it priority this week.

**Comment by ndelangen** [comment]
Hello @glrodasz It would be great to have to move forwards.

Please re-open if you're willing to continue to work on this.

You contribution is very very much appreciated, thank you for your effort, time and energy you've put into this PR.

**Pull request closed without merging**